plugins/generator-1.20.1/forge-1.20.1/templates/json/block_loot_table.json.ftl

```diff
@@ -0,0 +1,67 @@
+<#include "../mcitems.ftl">
+<#assign defaultSlabDrops = data.getModElement().getTypeString() == "block" && data.blockBase?has_content && data.blockBase == "Slab" && !(data.customDrop?? && !data.customDrop.isEmpty())/>
+{
+  "type": "minecraft:block",
+  "random_sequence": "${modid}:blocks/${registryname}"
+  <#if data.dropAmount != 0>,
+  "pools": [
+    {
+      "rolls": 1.0,
+      <#if data.dropAmount == 1 && !defaultSlabDrops>
+      "conditions": [
+        {
+          "condition": "minecraft:survives_explosion"
+        }
+      ],
+      </#if>
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": <#if data.customDrop?? && !data.customDrop.isEmpty()>"${mappedMCItemToRegistryName(data.customDrop)}"<#else>"${modid}:${registryname}"</#if>
+          <#if data.isDoubleBlock()>,
+          "conditions": [
+            {
+              "block": "${modid}:${registryname}",
+              "condition": "minecraft:block_state_property",
+              "properties": {
+                "half": "lower"
+              }
+            }
+          ]
+          </#if>
+          <#if data.dropAmount != 1>,
+          "functions": [
+            {
+              "count": ${data.dropAmount},
+              "function": "minecraft:set_count"
+            },
+            {
+              "function": "minecraft:explosion_decay"
+            }
+          ]
+          <#elseif defaultSlabDrops>,

```

Comment:
In case the drop amount is not 1 and block base is slabs, shouldn't this part also be loaded? Right now it would be not given elseif

Reply:
Currently double slab drops are doubled only if no custom loot is selected and count is 1. It could be changed so that drops are always doubled if it's a double slab

Reply:
I see, as long as it does not change current behavior, it is ok